engine/src/witness/common/chain_source/strictly_monotonic.rs

```diff
@@ -83,14 +83,15 @@ mod test {
 					Header { index: 4u32, hash: (), parent_hash: Some(()), data: () },
 					Header { index: 3u32, hash: (), parent_hash: Some(()), data: () },
 					Header { index: 2u32, hash: (), parent_hash: Some(()), data: () },
+					Header { index: 10u32, hash: (), parent_hash: Some(()), data: () },
```

Comment:
Is the addition of this to test case meaningful? How? (Would it have previously not passed?)

Reply:
It's not meaningful, it's just not obvious that it allows skipping blocks without that. Which informed me a little on how the rest of this would/could work